public/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": "com.github.jankaritech.web.mdpresentation",
```

Comment:
Needs suggestion (https://github.com/JankariTech/web-app-presentation-viewer/issues/31)

Reply:
:heart:

Reply:
id is not part of the manifest anymore, the id is now pulled from the folder name.

Reply:
I find it confusing to maintain the assets path used as id and app-id.
I can use appId `com.github.jankaritech.web.mdpresentation` and build app to `dist/coolapp` with `/assets/apps/coolapp/` as assets base path. Then again later while deploying I can rename the `dist/coolapp` to something like `dist/verycoolapp`. This doesn't affect the app loading but assets path will still be `/assets/apps/coolapp/`unless we build the app again with `/assets/apps/verycoolapp/` as assets base.